docs/recipes/auth/yarn.lock

```diff
@@ -0,0 +1,4702 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
we haven't really had a need to check in the yarn.lock files for the recipes, so this could probably be left out. They're big files and they don't really add anything to the readable example (and the package.json means things are still recreatable)